heareval/score.py

```diff
@@ -76,7 +99,12 @@ def __init__(
             self.name = name
         self.maximize = maximize
 
-    def __call__(self, predictions: Any, targets: Any, **kwargs) -> float:
+    def __call__(self, *args, **kwargs) -> float:
+        ret = self.compute(*args, **kwargs)
+        validate_score_return_type(ret)
```

Comment:
feels like the type signature is probably wrong here but I am not sure?
Have you run mypy by hand on heareval in this branch?

Reply:
fixed